bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -204,11 +210,10 @@ pub mod pallet {
 		#[transactional]
```

Comment:
Is the transactional macro here required?

Reply:
Probably not, will clean up later. :)